library.properties

```diff
@@ -6,5 +6,5 @@ sentence=Allows controlling the ST X-NUCLEO-IKS02A1 expansion board
 paragraph=This library provides several sample applications to control ST X-NUCLEO-IKS02A1 expansion board
 category=Sensors
 url=https://github.com/stm32duino/X-NUCLEO-IKS02A1
-architectures=stm32, avr, sam
-depends=STM32duino ISM330DHCX, STM32duino IIS2DLPC, STM32duino IIS2MDC
\ No newline at end of file
+architectures=stm32
```

Comment:
Hi @sarachiodo ,
because you put an error for the Audio example, you can restore the support for the other architecture in the library properties (stm32, avr, sam) as agreed with Frederic.
Best Regards,
Carlo